### PR TITLE
Form Submission

### PR DESCRIPTION
**Pull request by Jimmy-Tempest**
**Features**
- Form Submission now save to database

### CODE REVIEW

File: src/utils/backend/news/form.ts [hunk 1]
```diff
@@ -46,9 +52,108 @@ export async function getForm(
...
-): Promise<BackendReturn<FormQuestionsTable[]>> {
-  // TODO: Send form data to Supabase
+  formID: number,
+  formAnswer: { id: number; value: string | number | string[] | File | null }[],
+  sendAs?: number
+): Promise<
+  BackendReturn<
+    BackendReturn<FormFieldValueTable[] | FormFieldValueTable | null>[]
+  >
+> {
```
Comment (SiravitPhokeed) [inline src/utils/backend/news/form.ts:62]: I’m not sure what this means…?

**Review by SiravitPhokeed — APPROVED** [review]
LGTM!

File: src/utils/backend/news/form.ts (hunk 1 above)
Comment (Jimmy-Tempest) [reply src/utils/backend/news/form.ts:62]: Which part? as in what is this for or literally what is this code doing

Comment (SiravitPhokeed) [reply src/utils/backend/news/form.ts:62]: The return type. This part:
```ts
<
  BackendReturn<
    BackendReturn<FormFieldValueTable[] | FormFieldValueTable | null>[]
  >
>
```
Why are there 2 `BackendReturn`s?

Comment (Jimmy-Tempest) [reply src/utils/backend/news/form.ts:62]: Good Question

Comment (Jimmy-Tempest) [reply src/utils/backend/news/form.ts:62]: Fuck

**Review by SiravitPhokeed — APPROVED** [review]
LGTM!

**Pull request merged**